aioamqp_consumer/producer.py

```diff
@@ -6,11 +6,11 @@
 class Producer(AMQPMixin):
 
     def __init__(
-        self,
-        amqp_url,
-        *,
-        amqp_kwargs=None,
-        loop=None
+            self,
```

Comment:
why here is one more ident?

Reply:
Oh, PyCharm's fault... Didn't see this ugly autoformat issue.